Project/Source/Panels/PanelInspector.cpp

```diff
@@ -227,6 +230,28 @@ void PanelInspector::Update() {
 						App->editor->modalToOpen = Modal::COMPONENT_EXISTS;
 					}
 				}
+				if (ImGui::MenuItem("Particle")) {
+					ComponentParticleSystem* particle = selected->CreateComponent<ComponentParticleSystem>();
+					if (particle != nullptr) {
+						particle->Init();
+					}
```

Comment:
This is missing the else clause. It should be there even if we are allowing multiple particle systems, just in case we decide to not allow it.